### PR TITLE
Docs v2 Blog post

### PR DESCRIPTION
**Pull request by RamiroND**
New blog post.

### CODE REVIEW

File: apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: 'New Supabase Docs, built with Next.js'
+description: We've redesign our Docs and migrated to Next.js
+author: jonny, danny
+image: lw6-day-1/lw6-day1-docs.jpg
+thumb: lw6-day-1/lw6-day1-docs.jpg
+tags:
+  - launch-week
+date: '2022-12-12'
+toc_depth: 3
+---
```
Comment (awalias) [inline apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx:11]:
```suggestion
youtubeHero: https://www.youtube-nocookie.com/embed/Q1Amk6iDlF8 
---
```

File: apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+title: 'New Supabase Docs, built with Next.js'
+description: We've redesign our Docs and migrated to Next.js
+author: jonny, danny
```
Comment (awalias) [inline apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx:4]:
```suggestion
author: jonny
```

File: apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Previously when browsing through our reference docs, each function lived on a separate page. This release changes that behaviour - the new layout has all of a library’s functions on one page. This promotes discoverability within a single tool. Developers have the ability to use the search bar _or_ they can just `cmd+f` in their browser.
+
+We’ve also re-engineered how our reference docs are generated from typedoc, so it’s easier for contributors and our own team members to edit and add to Supabase documentation.
+
+Inspiration: [Stripe](https://stripe.com/docs/cli).
+
+## What’s next
+
+This is just the start. With this foundation, we hope to re-imagine what a documentation experience can be. Some things we’re working on:
```
Comment (kiwicopple) [inline apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx:70]:
```suggestion
With this foundation, we hope to re-imagine what a documentation experience can be. Some things we’re working on:
```

suggestion

File: apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Fully integrated experience
+
+- **Unified Authentication:** If you’re logged into the Dashboard, then you’re logged into Supabase docs. That means personalization and project-specific documentation. No more placeholders like “`SUPABASE_URL`". We will add URLs and Keys for the project you’ve selected, making it easier to copy and run code snippets.
+- **Auto docs:** We may also move AutoDocs (in the dashboard) into the main Docs website. Reference docs are then just, reference docs.. but you can see it in context to the table/schema your interested in 🤯.
```
Comment (kiwicopple) [inline apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx:75]: this seems a bit unedited?

Do we need the 🤯??

File: apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+We’ve started implementing a set of new UI components to help facilitate our growing documentation content. While working on these, we’ve started to also refine how we want to present this information with more care on the smaller things we'd overlooked before, such as panels, buttons and icons.
+
+![New reference docs](/images/blog/launch-week-6/day-1-new-docs/redesign.png)
+
+### Better navigation
+
+Landing on a documentation site for a product as large as Supabase can be daunting.
+There’s a lot to read. The problem is one of discoverability, which even a search interface can’t fix (how do you know what to search if you don’t know what a product can do?).
+
+To solve this, we built everything into one sidebar. From a glance, you can see everything we offer without clicking three-levels-deep into a maze of content. For a first-time reader, you should never feel lost in the docs.
```
Comment (kiwicopple) [inline apps/www/_blog/2022-12-12-new-supabase-docs-built-with-nextjs.mdx:54]:
```suggestion
To solve this, we built everything into a single sidebar. From a glance, you can see everything we offer without clicking three-levels-deep into a maze of content. For a first-time reader, you should never feel lost in the docs.
```

**Review by MildTomato — APPROVED** [review]

**Pull request merged**